### ISSUE TITLE
Add -y flag to zos init similar to npm

### ISSUE THREAD

**Issue by spalladino** [zeppelinos/zos#1034]
When setting up a new project, it's sometimes useful to skip the prompts and go straight to the action. Consider adding a `-y` flag similar to `npm init` that skips all questions.

**Comment by jcarpanelli** [comment]
Good idea! Also, if a `package.json` file has already been created, we can take the package name and version from there but if it's not, we can just set the project name as the directory name and set the initial version as `0.0.1` or `1.0.0` (I believe npm sets the latter as the default version)
